package.json

```diff
@@ -30,7 +30,11 @@
     "clean": "git clean -x -f",
     "compile": "tsc",
     "lint": "concurrently 'yarn:lint:*'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
```

Comment:
any reason to have these called automatically by the lint command? it doesn't seem to exclude :fix as the default ember bp does